multinode/storage/service.go

```diff
@@ -159,6 +163,10 @@ func (service *Service) TotalDiskSpace(ctx context.Context) (totalDiskSpace Disk
 	}
 
 	for _, node := range listNodes {
+		nodeinfo := service.nodes.FetchNodeInfo(ctx, node)
```

Comment:
The same that I commented before, we would also use here the function that checks the status of the node.